rules.js

```diff
@@ -45,4 +45,9 @@ export default [
   [/^transition-gpu$/, ([_]) => emitWarning(_, TYPES.removed, "use 'transform-gpu' or 'will-change-*'")],
   [/^fixed-ios-fix$/,([_]) => emitWarning(_, TYPES.removed, "use 'transform translate-z-0'")],
   [/^focus-ring$/,  ([_]) => emitWarning(_, TYPES.replaced, "use 'focusable'")],
+  // deprecated Warp classes
```

Comment:
Should we also add `^(.+)-active-hover$`? We have removed them and added `selected-hover` instead.

Reply:
Ah, good catch! Will add that 👍